notanorm/base.py

```diff
@@ -680,13 +680,25 @@ def create_model(self, model: DbModel, ignore_existing=False):
         for name, schema in model.items():
             self.create_table(name, schema, ignore_existing)
 
-    def create_table(
-        self, name, schema: DbTable, ignore_existing=False
-    ):  # pragma: no cover
-        raise RuntimeError("Generic create table not supported")
+    @abstractmethod
+    def create_table(self, name, schema: DbTable, ignore_existing=False):
+        ...
+
+    def create_indexes(self, name, schema: DbTable):
+        try:
+            existing = self.model()[name].indexes
+        except:
```

Comment:
[Remove bare except](https://stackoverflow.com/questions/54948548/what-is-wrong-with-using-a-bare-except). Don't think we want this to catch `KeyboardInterrupt`.